ui/xui/menubar.cc

```diff
@@ -69,6 +70,10 @@ void ProcessKeyboardShortcuts(void)
     if (ImGui::IsKeyPressed(ImGuiKey_F12)) {
         ActionScreenshot();
     }
+    if(ImGui::IsKeyPressed(ImGuiKey_C)) {
+        ExportData *data = NULL;
+        export_texture(data);
```

Comment:
You're passing in a null instance of the ExportData structure, so there will never be anything useful to export.